src/data/complex/module.lean

```diff
@@ -239,23 +239,19 @@ variables {A : Type*} [ring A] [algebra ℝ A]
 
 See `complex.lift` for this as an equiv. -/
 def lift_aux (I' : A) (hf : I' * I' = -1) : ℂ →ₐ[ℝ] A :=
-alg_hom.mk'
-  { to_fun := ⇑((algebra.of_id ℝ A).to_linear_map.comp re_lm +
-                (linear_map.to_span_singleton _ _ I').comp im_lm),
-    map_zero' := linear_map.map_zero _,
-    map_add' := linear_map.map_add _,
-    map_mul' := λ ⟨x₁, y₁⟩ ⟨x₂, y₂⟩, by {
-      change algebra_map ℝ A (x₁ * x₂ - y₁ * y₂) + (x₁ * y₂ + y₁ * x₂) • I' =
-             (algebra_map ℝ A x₁ + y₁ • I') * (algebra_map ℝ A x₂ + y₂ • I'),
+alg_hom.of_linear_map
+  ((algebra.of_id ℝ A).to_linear_map.comp re_lm + (linear_map.to_span_singleton _ _ I').comp im_lm)
+  (show algebra_map ℝ A 1 + (0 : ℝ) • I' = 1,
+    by rw [ring_hom.map_one, zero_smul, add_zero])
+  (λ ⟨x₁, y₁⟩ ⟨x₂, y₂⟩, show algebra_map ℝ A (x₁ * x₂ - y₁ * y₂) + (x₁ * y₂ + y₁ * x₂) • I'
+                          = (algebra_map ℝ A x₁ + y₁ • I') * (algebra_map ℝ A x₂ + y₂ • I'),
+    by {
```

Comment:
I was half thinking of this change in the previous PR and now that the `by` has ended up on its own line I am quite sure about it:
```suggestion
    begin
```